CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# How to contribute ?
+
+
+Are you ready to become a part of the eZ Community? There are several ways in which you can contribute, from spotting and reporting bugs to commenting on the documentation to discussing innovative uses on Slack to coding new bundles.
+We have a [Bug Tracker](https://jira.ez.no) and you can talk to a human on eZ Community Slack: https://ezcommunity.slack.com
+
+
+# Development Guidelines
+
+Our Development Guidelines are available on this page: https://doc.ez.no/display/DEVELOPER/Development+guidelines
+
+## Topics covered by this Guidelines
```

Comment:
I was thinking: you are offline, and you have to make the decision "_do you really need the Guidelines ?_"